### PR TITLE
Feature/22 place graphic elements introduction final pages

### PR DESCRIPTION
**Pull request by baxita**
Arrow made smaller, also moved arrow and reload under nth-of type( 1) under sections.

### CODE REVIEW

**Review by icanseetime — CHANGES_REQUESTED** [review]
Fix the commented suggestions. 
And another suggestion, but it's up to you is to name the ID's of the img's the same as the img files for consistency.

File: assets/styles/index.scss [hunk 1]
```diff
@@ -33,6 +33,24 @@ section {
 
   &:nth-of-type(1) {
     background: $gradient-pollutedsky-s1;
+
+    //introduction-page polluted arrow indicate scroll
+    //! TODO: add animation in later issue
+    & #arrow {
+      position: absolute;
```
[the remark this answers is not in the document]
Comment (baxita) [reply assets/styles/index.scss]: added suggested solution @include center

File: assets/styles/index.scss [hunk 2]
```diff
@@ -33,6 +33,24 @@ section {
...
+      top: 78%;
+      left: 48%;
+      width: 2.5vw;
+    }
+
+    //finalpage-dimmed reload-icon
+    //! TODO: add animation in later issue
+    & #reload {
+      position: absolute;
```
[the remark this answers is not in the document]
Comment (baxita) [reply assets/styles/index.scss]: added suggested solution @include center

**Review by icanseetime — APPROVED** [review]
Except for the random alt-tags, good job 🌟 We'll fix those in a later issue anyway.

**Comment by baxita** [comment]
> 
nice!
sure, I can fix it when doing accessibility issue !! 👍

**Pull request merged**